.golangci.next.reference.yml

```
@@ -529,6 +531,14 @@ linters:
       # Default: false
       analyze-types: true
 
+    funcorder:
+      # Enable/disable feature to check constructors are placed after struct declaration
```

Please add `.` at the end for consistency:
```suggestion
      # Enable/disable feature to check constructors are placed after struct declaration.
```

---

Done!